ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/switch/v1/switch/switch.html

```diff
@@ -37,14 +37,15 @@
         <div data-sly-call="${label.label @componentId=widgetId, labelValue=switch.label.value, labelVisible=switch.label.visible, bemBlock='cmp-adaptiveform-switch'}" data-sly-unwrap></div>
     </div>
 
-    <label class="cmp-adaptiveform-switch__label-on">${switch.enumNames[0]}</label>
-    <label class="cmp-adaptiveform-switch__widget1">
+    <label>${switch.enumNames[0]}</label>
```

Comment:
Not clear why the BEM has been removed. Without it - we cannot style it individually